README.md

```diff
@@ -108,6 +114,8 @@ Copy the following files to `files/lib` directory.
 1. [MySQL Connector/J](https://dev.mysql.com/downloads/connector/j/5.1.html)
 2. [Amazon Corretto for Linux x64 JDK](https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/downloads-list.html)
 
+Copy the miscellaneous files to `files/misc` directory.
```

Comment:
Shall we also mention that `misc_file_list` for the respective roles should be uncommented and the `src` and `dest` values should be set?

Reply:
+1

Reply:
Fixed with https://github.com/wso2/ansible-apim/pull/70/commits/35cd23cf3403ed75f4c3a476f69d0ea68bb23781